README.md

```diff
@@ -97,6 +126,31 @@ require('swenv').setup({
 })
 ```
 
+### Reload LSP client on setting venv
+
+You can get your lsp client by name and reload the client after setting the venv.
+
+```lua
+post_set_venv = function()
+  local client = vim.lsp.get_clients({ name = "basedpyright" })[1]
+  if not client then
+    return
+  end
+  local venv = require("swenv.api").get_current_venv()
+  if not venv then
+    return
+  end
+  local venv_python = venv.path .. "/bin/python"
+  if client.settings then
+    client.settings = vim.tbl_deep_extend("force", client.settings, { python = { pythonPath = venv_python } })
+  else
+    client.config.settings =
+        vim.tbl_deep_extend("force", client.config.settings, { python = { pythonPath = venv_python } })
+  end
+  client.notify("workspace/didChangeConfiguration", { settings = nil })
```

Comment:
just out of curiosity, I don't set `python.pythonPath` which just uses whatever `python` is in my current environment and use `LspRestart`. Is there any advantage or your approach here?

Reply:
An example of both might indeed make sense, or at least describe the options with their pros and cons?